tests/steps/facebook.coffee

```diff
@@ -1,6 +1,6 @@
 module.exports = ->
   @Given /^I create a Facebook Share Button$/, () ->
-    @driver.visit("http://localhost:8000/tests/fixtures/facebook.html")
+    @driver.visit("file:///#{process.cwd()}/tests/fixtures/facebook.html")
```

Comment:
This is actually suppose to be localhost:8000 because of how the sdk gets loaded - it was an issue we ran into